ako-infra/avirest/networking_handler.go

```diff
@@ -87,10 +87,7 @@ func AviCloudCachePopulate(client *clients.AviClient, cloudName string, skipIPAM
 		return err
 	}
 
-	if !skipIPAM {
-		AviIPAMCachePopulate(client, strings.Split(*CloudCache.IPAMProviderRef, "#")[1])
-	}
-	return nil
+	return AviIPAMCachePopulate(client, strings.Split(*CloudCache.IPAMProviderRef, "#")[1])
```

Comment:
this will result in crash if IPAM is not configured in cloud

Reply:
IPAM configuration is part of initial setup required for enabling WCP. I had added skipIPAM flag for VPC case which isn't required now. Anyway, I can check for IPAM in cloud and shutdown AKO if IPAM is not present.

Reply:
Yes, handle it gracefully.